pkg/tlsclientconfig/tlsclientconfig.go

```diff
@@ -44,6 +44,10 @@ func SetupCertificates(dir string, tlsc *tls.Config) error {
 			logrus.Debugf(" crt: %s", fullPath)
 			data, err := ioutil.ReadFile(fullPath)
 			if err != nil {
+				if os.IsNotExist(err) {
```

Comment:
\*shrug\* there is a reasonable case for the new behavior, but please at least add a comment above this noting that this can happen if `fullPath` is a dangling symlink (or during a race against something removing a file from the directory?); otherwise, because `fullPath` was created by listing the parent directory, it’s not immediately obvious how this can happen.

Reply:
Added a comment mentioning both possibilities.